packages/office-ui-fabric-react/src/components/Checkbox/Checkbox.styles.ts

```diff
@@ -54,7 +54,12 @@ export const getStyles = (props: ICheckboxStyleProps): ICheckboxStyles => {
       borderColor: disabled ? checkboxBorderColorDisabled : checkboxBorderIndeterminateColor,
       transitionProperty: 'border-width, border, border-color',
       transitionDuration: MS_CHECKBOX_TRANSITION_DURATION,
-      transitionTimingFunction: MS_CHECKBOX_TRANSITION_TIMING
+      transitionTimingFunction: MS_CHECKBOX_TRANSITION_TIMING,
+      selectors: {
+        [HighContrastSelector]: {

```

Comment:
Just to clarify, this is keeping the use of high contrast selectors to avoid a breaking change while adding the usage of the new css selector `forced-color-adjust` as a better alternative option?